app/query/caller_test.go

```diff
@@ -401,11 +413,272 @@ func TestSDKMethodStatus(t *testing.T) {
 		rpcResponse.Result.(map[string]interface{})["installation_id"].(string))
 }
 
-func parseRawResponse(t *testing.T, rawCallResponse []byte, v interface{}) {
-	assert.NotNil(t, rawCallResponse)
-	var res jsonrpc.RPCResponse
-	err := json.Unmarshal(rawCallResponse, &res)
+func TestCaller_GetPaidCannotPurchase(t *testing.T) {
+	dummyUserID := rand.Intn(99999)
+	srvAddress := test.RandServerAddress(t)
+	uri := "lbry://@specialoperationstest#3/iOS-13-AdobeXD#9"
+	err := wallet.Create(srvAddress, dummyUserID)
+
+	request := jsonrpc.NewRequest(MethodGet, map[string]interface{}{"uri": uri})
+	rawCallResponse := NewCaller(srvAddress, dummyUserID).Call(request)
+
+	var errorResponse jsonrpc.RPCResponse
+	err = json.Unmarshal(rawCallResponse, &errorResponse)
 	require.NoError(t, err)
-	err = res.GetObject(v)
+	fmt.Println(string(rawCallResponse))
+	assert.Equal(t, "Not enough funds to cover this transaction.", errorResponse.Result.(map[string]interface{})["error"])
+}
+
+func TestCaller_GetPaidPurchased(t *testing.T) {
+	config.Override("BaseContentURL", "https://cdn.lbryplayer.xyz/api/v2/streams/")
```

Comment:
every time i see a `config.Override()` in a test, it makes me think that that value should be passed in to a `NewX()` func during initialization rather gotten directly from the global config. do you feel the same way?

Reply:
It does feel like that. But I guess compromises have to be made. This variable is just used once, to promote it to an initializer parameter feels wasteful.